tests/unity/CMakeLists.txt

```diff
@@ -198,9 +204,11 @@ endfunction()
 #for example if file under test is include mocked header as <foo/header.h> then
 # mock and replaced header should be placed in <mock_path>/foo with <mock_path>
 # added as include path.
-# EXCLUDE <pattern>: Exclude functions matching pattern. Pattern can be a simple style regex.
+# FUNC_EXCLUDE <pattern>: Exclude functions matching pattern. Pattern can be a simple style regex.
```

Comment:
unsure if customers are using those functions, but should we just deprecated old EXCLUDE and print a deprecated warning if used, and then treat EXCLUDE as FUNC_EXCLUDE for a couple of releases ?

Feel free to comment.

Reply:
Makes sense, updated.